frontend/src/app/components/wp-table/configuration-modal/tabs/columns-tab.component.ts

```diff
@@ -1,24 +1,21 @@
-import {AfterViewInit, Component, ElementRef, Injector, OnDestroy, ViewChild} from '@angular/core';
+import {Component, Injector, OnInit} from '@angular/core';
 import {I18nService} from 'core-app/modules/common/i18n/i18n.service';
 import {QueryColumn} from 'core-components/wp-query/query-column';
 import {ConfigurationService} from 'core-app/modules/common/config/configuration.service';
 import {WorkPackageViewColumnsService} from 'core-app/modules/work_packages/routing/wp-view-base/view-services/wp-view-columns.service';
 import {TabComponent} from 'core-components/wp-table/configuration-modal/tab-portal-outlet';
 import {BannersService} from "core-app/modules/common/enterprise/banners.service";
-
-interface ColumnLike {
-  text:string;
-  id:string;
-}
+import {DraggableOption} from "core-app/modules/common/draggable-autocomplete/draggable-autocomplete.component";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/6ebea69939d32d037236e691655ea517c0401be9/frontend/src/app/components/wp-table/configuration-modal/tabs/columns-tab.component.ts#L9"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>